src/gamma.jl

```diff
@@ -132,12 +141,12 @@ const cotderiv_Q = [cotderiv_q(m) for m in 1:100]
 function cotderiv(m::Integer, z)
     isinf(imag(z)) && return zero(z)
     if m <= 0
-        m == 0 && return π * cot(π*z)
+        m == 0 && return π * _cotpi(z)
```

Comment:
It seems codecov complains about this line (and the following one). Maybe we could add a test for it and move it before the `if` statement? IMO this could be left for a separate PR though.